app/assets/stylesheets/application.css.scss

```diff
@@ -54,121 +54,3 @@ a:hover, a:active, a:visited, a:focus {
   text-decoration: underline;
 }
 // End duplicated styles
-
-// accessible color contrast
-.btn-danger {
-  background-color: #A42823 !important;
-  border-color: #A42823 !important;
-}
-
-.btn-primary {
-  @extend .btn-primary;
-  background-color: $daria-color;
-  border: none;
-  color: white;
-  margin-bottom: 2px;
-  font-size: $font-size;
-}
-
-.btn-primary:hover {
-  background-color: lighten($daria-color,15%);
-}
-
-.btn-primary:focus {
-  background-color: lighten($daria-color,15%);
-  color: white;
-  text-decoration: none;
-}
-
-
-
-
-.btn-lg {
-  margin: 15px 0;
-  padding: 15px 30px;
-}
-
-
-.button-xs {
-// maybe deleteable?
-  @extend .btn;
-  @extend .btn-primary;
-  @extend .btn-sm;
-}
-
-
-$daria-color-lt : #825fb9;
-
-.btn-success {
-  background-color: $daria-color;
-  border-color: $daria-color;
-
-  &:hover {
-    background-color: $daria-color-lt;
-    border-color: $daria-color;
-  }
-  &:active {
-    background-color: #286090 !important;
-  }
-  &:focus {
-    background-color: $daria-color;
-    &:hover {
-    background-color: $daria-color-lt;
-    }
-  }
-}
-
-.btn-warning {
-  color: black;
-  border-color: $daria-color;
-  background-color: white;
-  &:hover {
-    background-color: $daria-color-lt;
-    border-color: $daria-color;
-  }
-  &:active {
-    background-color: #286090 !important;
-  }
-  &:focus {
-    background-color: $daria-color;
-    &:hover {
-    background-color: $daria-color-lt;
-    }
-  }
-}
-
-@mixin disabled-button {
-  color: white;
-  background-color: gray;
-  border-color: $daria-color;
-  &:hover {
-    background-color: gray;
-    border-color: $daria-color;
-  }
-  &:focus {
-    background-color: gray;
-    border-color: $daria-color;
-    &:hover {
-      background-color: gray;
-      border-color: $daria-color;
-    }
-  }
-}
-
-.btn-success[disabled] {
-  @include disabled-button;
-}
-
-.btn-warning[disabled] {
-  @include disabled-button;
-}
-
-
-.btn[data-orientation='cancel'] {
-  color: red;
-}
-.submit {
-  @extend .btn;
-  @extend .btn-primary;
-  @extend .btn-lg;
-  }
```

Comment:
Did we mean to delete this one?

Reply:
yep! This is an unused class, so we can remove the styling guilt free.